django_dramatiq/apps.py

```diff
@@ -44,12 +44,22 @@ def import_models(self):
         that Dramatiq is fully configured before that happens.
         """
         super().import_models()
-        self.ready()
+        self.initialize()
 
     def ready(self):
         if getattr(self, "_is_ready", False):
             return
         super().ready()
```

Comment:
Having this piece in place would mean that we will never call `ready()` method, as here `initialize` is setting `_is_ready=True`. This generally won't be correct behaviour, even if it may solve the issue with multiple `ready()` call.


Would you mind trying out https://github.com/Bogdanp/django_dramatiq/pull/137 this solution to see if this changes things for you?

Best,
Rust